src/cluster_legacy.c

```diff
@@ -1145,6 +1155,16 @@ static clusterMsgSendBlock *createClusterMsgSendBlock(int type, uint32_t msglen)
     return msgblock;
 }
 
+static clusterMsgSendBlockLight *createPubsubClusterMsgSendBlock(int type, uint32_t msglen) {
+    uint32_t blocklen = msglen + sizeof(clusterMsgSendBlockLight) - sizeof(clusterMsgLight);
+    clusterMsgSendBlockLight *msgblock = zcalloc(blocklen);
+    msgblock->refcount = 1;
+    msgblock->totlen = blocklen;
+    server.stat_cluster_links_memory += blocklen;
+    clusterBuildPubsubMessageHdr(&msgblock->msg, type, msglen);
+    return msgblock;
+}
```

Comment:
This also seems to be unrelated to pubsub. Maybe rename to `createClusterLightMsgSendBlock` ?

Reply:
Removed this function in order to reduce duplication